### PR TITLE
Tail rom

### PR DESCRIPTION
**Pull request by alanvgreen**

### CODE REVIEW

**Comment by alanvgreen** [comment]
There's going to be a fair amount of work in this stream, so wanted to get something out for review sooner rather than later.

File: common/src/tflite.cc [hunk 1]
```diff
@@ -171,6 +171,10 @@ void tflite_load_model(const unsigned char* model_data) {
     return;
   }
 
+#ifdef TF_LITE_SHOW_MEMORY_USE
+  interpreter->GetMicroAllocator().PrintAllocations();
+#endif
```
Comment (tcal-x) [inline common/src/tflite.cc:179]: PrintAllocations() provides the info from arena_used_bytes() and much more, correct?

**Review by tcal-x — APPROVED** [review]
LGTM

File: common/src/tflite.cc (hunk 1 above)
Comment (alanvgreen) [reply common/src/tflite.cc:179]: Yes, that's right.

**Pull request merged**